src/contracts/nft-drop.ts

```diff
@@ -109,7 +109,7 @@ export class NFTDrop extends Erc721<DropERC721> {
    * await dropContract.claimConditions.set(claimConditions);
    * ```
    */
-  public claimConditions: DropErc721ClaimConditions;
+  public claimConditions: DropErc721ClaimConditions<DropERC721>;
```

Comment:
since this is a generic type now do we still need the specific `DropErc721ClaimConditions` name? or can it become a generic `ClaimConditions<DropErc721>` ?

Reply:
Yep, just hadn't renamed yet. Still some things to work out in this pr